packages/cxl-ui/scss/global/cxl-card.scss

```diff
@@ -0,0 +1,69 @@
+cxl-card {
+  box-sizing: border-box;
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  margin: var(--lumo-space-m);
+  overflow: hidden;
+  font-family: inherit;
+  text-overflow: ellipsis;
+  background: var(--lumo-base-color);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius);
+
+  &:hover {
+    border-color: var(--lumo-primary-color);
+  }
+
+  iron-icon {
+    --iron-icon-height: var(--lumo-icon-size-s);
+    --iron-icon-width: var(--iron-icon-height);
+  }
+
+  .entry-header {
+    margin: 0;
+    font-family: inherit;
+    font-style: normal;
+
+    .headshot {
+      width: 120px;
+      height: 120px;
+      margin: 0 auto;
+      border: 5px solid var(--lumo-base-color);
+      border-radius: 100%;
+    }
+
+    .company-logo {
+      max-width: 100px;
+      margin: var(--lumo-space-s) auto;
+    }
+
+    .card-name,
+    h4 {
+      padding: 0;
+      margin: 0;
+    }
+
+    .card-description,
+    p {
+      margin: 0 auto;
+      font-family: inherit;
+      font-size: var(--lumo-font-size-s);
+    }
+  }
+
+  .entry-summary {
+    padding: 0 var(--lumo-space-s);
+
+    p {
+      font-family: inherit;
+    }
+  }
+
+  &[theme~="cxl-testimonial"] {
+    width: 400px;
```

Comment:
What's cxl-accordion-card width strategy? We should follow it - meaning, potentially let the grid define it by column width.

Reply:
Since the container isn't constrained by width, column-width won't work here. I used `width: calc(var(--cxl-wrap-width) / 2.5);` in the story.